protos/escrow/escrow.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+package escrow;
+
+// golang option
+option go_package = "escrowpb";
+// java options
+option java_multiple_files = true;
+option java_outer_classname = "EscrowProto";
+option java_package = "io.btfs.escrow";
+
+import "google/protobuf/timestamp.proto";
+
+service EscrowService {
```

Comment:
Can you add comment on each RPC function? Describe who should call and what to expect to return. For example, obly authority is valid to call CancelContracts. Comments would help new hire quickly understands the whole.

Reply:
OK